build/rhel-build-service/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# ARG BUILD_IMAGE=golang:1.11.5
+# ARG BASE_IMAGE=storageos/base-image:0.1.0
+# ARG OPERATOR_IMAGE=storageos/cluster-operator:test
+
+FROM golang:1.11.5 AS build
+# OPERATOR_IMAGE needs to be passed to build/cluster-operator for constructing
+# the ldflags.
+# ARG OPERATOR_IMAGE
+WORKDIR /go/src/github.com/storageos/cluster-operator/
+COPY . /go/src/github.com/storageos/cluster-operator/
+RUN make operator-sdk
+RUN make generate
+# RUN make build/cluster-operator OPERATOR_IMAGE=$OPERATOR_IMAGE
+RUN make build/cluster-operator OPERATOR_IMAGE=registry.connect.redhat.com/storageos/cluster-operator:1.0.0
+RUN make build/upgrader
+
+FROM storageos/base-image:0.1.0
```

Comment:
Hard-coded image names because ARG default values don't render in rhel build service. It works fine with the latest docker.